.cirrus.yml

```diff
@@ -41,6 +41,9 @@ env:
   IRIS_TEST_DATA_VERSION: "2.5"
   # Base directory for the iris-test-data.
   IRIS_TEST_DATA_DIR: ${HOME}/iris-test-data
+  # https://github.com/numpy/numpy/pull/19478
+  # https://github.com/matplotlib/matplotlib/pull/22099
+  NPY_DISABLE_CPU_FEATURES: "AVX512F,AVX512CD,AVX512VL,AVX512BW,AVX512DQ,AVX512_SKX"
```

Comment:
This is just horrible.

But it's necessary to disable this CPU feature to ensure that the tests running on Cloud CI platforms behave the same as local host runs; provided the local host doesn't also have Intel Short Vector Math Library (SVML) support for the platform chipset (in which case, also disable it)

Reply:
Might be best to push this down into the `noxfile` instead...

Reply:
Done.